ts/live-compositor/src/components/Text.ts

```diff
@@ -93,8 +93,8 @@ function sceneBuilder(props: TextProps, children: SceneComponent[]): Api.Compone
     max_height: style?.maxHeight,
     font_size: style?.fontSize ?? DEFAULT_FONT_SIZE,
     line_height: style?.lineHeight,
-    color_rgba: style?.color && intoApiRgbaColor(style?.color),
-    background_color_rgba: style?.backgroundColor && intoApiRgbaColor(style?.backgroundColor),
+    color: style?.color && intoApiRgbaColor(style?.color),
```

Comment:
is the change in sdk added by mistake or intentional? I'm fine either way but keep in mind that you will have to use binaries build from source and point `LIVE_COMPOSITOR_PATH` to that binary.

If you build with `cargo build --release --no-default-features` then executable is in `./target/release/main_process`

(no default features is important to build without chrome support)

Reply:
also after this change you don't need intoApiRgbaColor

Reply:
leftover, sorry for that